app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+use App\Models\Task;
+
+class TaskController extends Controller
+{
+    //
+    function displayTasks() { 
```

Comment:
rename function name to `display` as we are in `TaskController` so we dont need to add word `task` in function name